assets/indexCtrl.js

```diff
@@ -114,7 +107,7 @@ module.exports = function ($scope, io) {
         if (data && data.err) {
           $scope.addRefError = data.err;
         } else {
-          $scope.addRefError = "Already added that URL.";
+          $scope.addRefError = "There was an error.";
```

Comment:
Ah yes, thanks. We should probably give some more useful errors than this, but at least this isn't misleading.